examples/simple_box/src/main.rs

```diff
@@ -77,27 +81,52 @@ fn run(settings: Settings, cli: Cli) {
+            let client_net_config = build_client_netcode_config(
                 client_id.unwrap_or(settings.client.client_id),
                 // when communicating via channels, we need to use the address `LOCAL_SOCKET` for the server
                 LOCAL_SOCKET,
                 settings.client.conditioner.as_ref(),
                 &settings.shared,
                 transport_config,
             );
-            let mut client_app = client_app(settings.clone(), net_config);
-
-            // create server app
-            let extra_transport_configs = vec![TransportConfig::Channels {
+            let server_extra_transport_configs = vec![TransportConfig::Channels {
                 // even if we communicate via channels, we need to provide a socket address for the client
                 channels: vec![(LOCAL_SOCKET, to_server_recv, from_server_send)],
             }];
-            let mut server_app = server_app(settings, extra_transport_configs);
-
-            // run both the client and server apps
-            std::thread::spawn(move || server_app.run());
-            client_app.run();
+            let mut app = combined_app(settings, server_extra_transport_configs, client_net_config);
+            app.run();
         }
+        // #[cfg(not(target_family = "wasm"))]
+        // Cli::ListenServer { client_id } => {
+        //     // create client app
+        //     let (from_server_send, from_server_recv) = crossbeam_channel::unbounded();
+        //     let (to_server_send, to_server_recv) = crossbeam_channel::unbounded();
+        //     // we will communicate between the client and server apps via channels
+        //     let transport_config = TransportConfig::LocalChannel {
+        //         recv: from_server_recv,
+        //         send: to_server_send,
+        //     };
+        //     let net_config = build_client_netcode_config(
+        //         client_id.unwrap_or(settings.client.client_id),
+        //         // when communicating via channels, we need to use the address `LOCAL_SOCKET` for the server
+        //         LOCAL_SOCKET,
+        //         settings.client.conditioner.as_ref(),
+        //         &settings.shared,
+        //         transport_config,
+        //     );
+        //     let mut client_app = client_app(settings.clone(), net_config);
+        //
+        //     // create server app
+        //     let extra_transport_configs = vec![TransportConfig::Channels {
+        //         // even if we communicate via channels, we need to provide a socket address for the client
+        //         channels: vec![(LOCAL_SOCKET, to_server_recv, from_server_send)],
+        //     }];
+        //     let mut server_app = server_app(settings, extra_transport_configs);
+        //
+        //     // run both the client and server apps
+        //     std::thread::spawn(move || server_app.run());
+        //     client_app.run();
+        // }
```

Comment:
No need for this anymore!

Reply:
I think I might still keep it as an example; maybe
`ListenServer` -> 2 apps 
`Unified` -> 1 app with both client and server

What do you think of the naming?

Reply:
Hmm yeah naming is hard. Unified/segregated might be a good split? I don't really know